CefSharp.BrowserSubprocess.Core/Async/JavascriptAsyncObjectWrapper.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright � 2010-2015 The CefSharp Project. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#include "stdafx.h"
+#include "JavascriptCallbackRegistry.h"
+#include "JavascriptAsyncObjectWrapper.h"
+#include "JavascriptAsyncMethodWrapper.h"
+
+using namespace System::Linq;
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        namespace Async
+        {
+            void JavascriptAsyncObjectWrapper::Bind(const CefRefPtr<CefV8Value> &value)
+            {
+                //V8Value that represents this javascript object - only one per complex type, no accessor
+                auto javascriptObject = value->CreateObject(nullptr);
```

Comment:
From memory `CreateObject` is a static method, so prefer to have it called as such.